components/orderForm/OrderDetails.js

```diff
@@ -37,6 +40,12 @@ export default function OrderDetails({children}) {
       return
     }
 
+    if (inventoryInfo && inventoryInfo.markdown) {
+      inventoryInfo = inventoryInfo.markdown.split(/(?=####)/g)
+
```

Comment:
could you delete this empty line? just makes things a tiny bit shorter

Reply:
not really sure where this line is anymore because i moved it... i think i deleted it when i moved it?